docs/content/develop/step-5.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "Publishing Releases"
+description = ""
+weight = 10
+draft = false
+bref = ""
+toc = true
+menu = "develop"
+tags = [ "develop" ]
++++
+
+
+## Release Tag, Docs, & Storybook Updates
+
+
+### I. Get the latest from master locally, clean up
+
+
+
+1. `git status`
+    1. Check where you are
+2. `git checkout master && git fetch origin && git pull origin`
+    2. Grab master from origin
+3. `npm run clean`
+    3. Wipes node_modules directory and deletes any components not in master
+    4. what this does:
+    5. rm -rf node_modules package-lock.json
+    6. rm -rf node_modules
+4. `git reset --hard && git clean -df`
+    7. Remove any non-committed files, crufty directories
+5. `npm install`
+    8. Installs all dependencies fresh
+6. `git checkout -b temp`
+    9. Create temp branch
+
+
+### III. Update (changelog_) _and docs
+
+
+
+7. `git log --oneline --decorate --no-merges v1.0.0-prerelease.20`
+    10. See the diffs of what was merged between now & the last release
+    11. Or look at PR merged list with "last-updated" sort order: \
+[https://github.com/patternfly/patternfly-elements/pulls?q=is%3Apr+is%3Aclosed+sort%3Aupdated-desc](https://github.com/patternfly/patternfly-elements/pulls?q=is%3Apr+is%3Aclosed+sort%3Aupdated-desc)
+8. Update the file `CHANGELOG-prerelease.md` and save
+9. <span style="text-decoration:underline;">Search</span> for release tag string in /docs/layout directory i.e. "prerelease.19" and update to release number \
+
+<img src="/version_storybook.png" width="450"/>
+
+10. `git add * && git commit -m "Changelog, bump version in docs"`
+    12. Commit changes, don't push
+
+
+### IV. Create release branch & tag
+
+1. `npm run release`
+
+    This script will do the following:
+
+    1. Create release branch
+    2. Create compiled assets and commit them
+    3. Uses learna to guess at what the next release will be
+        1. **Choose "custom prerelease" (for now)**
+        2. It bumps the version numbers inside the package.json files in the components (currently all components are bumped, will fix)
+    4. Pushes the branch
+    5. Pushes independent tags for each component and pushes to NPM
+    6. Remove compiled assets
+    7. Returns you to the master branch
+
+
+### V. Github Updates
+
+
+
+1. Create a PR
+2. Get someone to review 
+3. Merge the release branch [pull request](https://github.com/patternfly/patternfly-elements/pulls) into master
```

Comment:
It would be good to note here that release PRs must never be squashed.

```suggestion
3. Merge the release branch [pull request](https://github.com/patternfly/patternfly-elements/pulls) into master.  Note: this must be a regular merge.  _Do not squash!_
```